tests/wdio.conf.ts

```diff
@@ -240,6 +245,17 @@ export const config: WebdriverIO.MultiremoteConfig = {
                 globalAny.ctx.webhooksProxy.connect();
             }
         }
+
+        if (testProperties.useWebhookProxy && !globalAny.ctx.webhooksProxy) {
+            console.warn(`WebhookProxy is not available, skipping ${testName}`);
+            globalAny.ctx.skipSuiteTests = true;
+        }
+
+        if (testProperties.useJaas && !globalAny.ctx.isJaasAvailable) {
+            console.warn(`JaaS is not configured, skipping ${testName}. `
+                + 'Set JAAS_DOMAIN, JAAS_TENANT, JWT_KID, and JWT_PRIVATE_KEY_PATH to enable.');
```

Comment:
So JAAS_TENANT overwrites the supplied IFRAME_TENANT? 
https://github.com/jitsi/jitsi-meet/blob/5b7e3bb2d77bfb9c795d2a20d5e4a2bf7ce54b8e/tests/env.example#L25

JAAS_TENANT is missing from env.example where we add all env variables we use with docs, examples and default value if applicable.